app/dialog_window.py

```diff
@@ -7,12 +7,32 @@ def __init__(self, app):
         self.field = app.field
         self.pause_the_game()
         super().__init__()
+        self.withdraw()

```

Comment:
What is the purpose of this call?

Reply:
The purpose is to hide the window until the program figures out it's geometry.